examples/counter-postgres/config/default.toml

```diff
@@ -5,9 +5,19 @@ snapshot-after = 100_000_000
 
 [evt-log]
 host          = "localhost"
-port          = 5432
-user          = "test"
-password      = "test"
+port          = 55000
```

Comment:
If you install postgres on mac using docker, this is the defaults.

Reply:
Hmm ... but if you install it on a Mac using `brew`, `5432` is the port. I guess I have to document the examples better and explain how to override the configuration settings, e.g. via env vars. In this case it would be `APP__EVT_LOG__PORT`.